src/Aspire.Hosting.Sdk/Aspire.RuntimeIdentifier.Tool/Aspire.RuntimeIdentifier.Tool.csproj

```
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <RollForward>Major</RollForward>
+    <UsePublicApiAnalyzers>false</UsePublicApiAnalyzers>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
```

We probably want to disable the app host `app.exe` file since we are never going to use it, and it won't work on RIDs that aren't what we built this for.

```suggestion
    <UseAppHost>false</UseAppHost
  </PropertyGroup>
```